staging/src/k8s.io/csi-translation-lib/plugins/azure_file.go

```diff
@@ -72,6 +72,7 @@ func (t *azureFileCSITranslator) TranslateInTreeInlineVolumeToCSI(volume *v1.Vol
 		Spec: v1.PersistentVolumeSpec{
 			PersistentVolumeSource: v1.PersistentVolumeSource{
 				CSI: &v1.CSIPersistentVolumeSource{
+					Driver:           AzureFileDriverName,
```

Comment:
I think we should also add `Driver:           AzureFileDriverName,` in Line 102

Reply:
indeed, nice catch. done